### PR TITLE
Knox topology fixes for various services

### PR DESCRIPTION
**Pull request by keyki**

### CODE REVIEW

File: orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2 [hunk 1]
```diff
@@ -201,7 +171,7 @@
     <service>
         <role>ZEPPELINWS</role>
         {% for hostloc in salt['pillar.get']('gateway:location')['ZEPPELIN_SERVER'] -%}
-        <url>ws://{{ hostloc }}:9995/ws</url>
+        <url>ws://{{ hostloc }}:{{ ports['ZEPPELIN'] }}</url>
```
Comment (risdenk) [inline orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2]: Need to keep the `/ws` on the end of this. Sorry didn't get to review this quicker.

Comment (risdenk) [reply orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2]: It should be `<url>ws://{{ hostloc }}:{{ ports['ZEPPELIN'] }}/ws</url>`

Comment (keyki) [reply orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2]: Hm, I've tested this and worked without /ws

Comment (keyki) [reply orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2]: I'll double check.

Comment (risdenk) [reply orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2]: @keyki and I checked this and looks like I was wrong and don't need the `/ws`.

**Pull request merged**